aioauth/collections.py

```diff
@@ -33,8 +34,24 @@ class HTTPHeaderDict(UserDict):
         d['hElLo'] == 'world' # >>> True
     """
 
+    def __init__(self, dict=None, /, **kwargs):
```

Comment:
could you remove this method signature? otherwise it will not work with older versions of Python:

```suggestion
    def __init__(self, dict=None, **kwargs):
```